### ISSUE TITLE
no suitable peers 

### ISSUE THREAD

**Issue by nickkliesen** [ethereum/mist#3889]
* [ y] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
windows 10
Node version: `geth 0.0.0`
Number of blocks synchronized:5555267
```

I can not get my wallet to sync it tells me that there is no suitable peers I ave included a screen shot of the log. I can not get to my coins. but they are there on ether scan
![logfile](https://user-images.githubusercontent.com/37019916/39968083-57201660-5695-11e8-8ed5-78d5842a52d4.PNG)

**Comment by stbregeon** [comment]
Hello,
I have the same problem since this morning

**Comment by stbregeon** [comment]
@ryanio Hello !
Have you got somes news about this problemes ??
thanks

**Comment by ryanio** [comment]
Hey guys sorry about the issues, we are working on a better node management system in development right now so you can use a remote node on start, then switch to local when sync'd up.

The no suitable peers is probably because there are not enough light servers serving light clients. You can also try switching to 'Fast' sync mode in the menu (under Develop).

Until the new version is ready, you can use [MetaMask](https://metamask.io/) and [MyEtherWallet.com](https://myetherwallet.com/) or [MyCrypto.com](https://mycrypto.com/) to access your accounts and funds.

**Comment by stbregeon** [comment]
@ryanio 
thanks a lot for all theses informations.

yesterday, I had to restore the last backup on Friday to solve the problems of peers not suitable.

my wallet is working again : I can connect to the network 

i had always use syn with lignt client.

but effectively the peers suddenly became rare unlike last weekes !

**Comment by tocorobo** [comment]
@ryanio I'm not finding 'Fast' sync mode under the Develop menu in v0.10.0.

I see only this option to toggle light client on or off:
Develop>Sync with Light client (beta)

This is on Windows 10

**Comment by onekiloparsec** [comment]
No 'Fast' sync mode in the Develop menu either here (macOS High Sierra), Wallet v0.10.

Wairing for the new version...

**Comment by ryanio** [comment]
Hello all, we released our new version (`0.11.0`) with new syncing features - please give it a try! If you have any new problems that you can reproduce, please create a new issue with the details on how 
we can help. Thanks!
